content/docs/reference-react.md

```diff
@@ -124,11 +124,11 @@ const MyComponent = React.memo(function MyComponent(props) {
 });
 ```
 
-`React.memo` is a [higher order component](/docs/higher-order-components.html). It's similar to [`React.PureComponent`](#reactpurecomponent) but for function components instead of classes.
+`React.memo` merupakan [higher order component](/docs/higher-order-components.html). Hal ini mirip dengan [`React.PureComponent`](#reactpurecomponent) tapi digunakan untuk _function component_ daripada _class component_.
 
-If your function component renders the same result given the same props, you can wrap it in a call to `React.memo` for a performance boost in some cases by memoizing the result. This means that React will skip rendering the component, and reuse the last rendered result.
+Apabila _function component_ anda me-_render_ hasil yang sama dengan _props_ yang sama juga, anda dapat membungkusnya dengan menggunakan `React.memo` untuk meningkatkan kinerjanya dengan cara mengingat (_memorizing_) hasil _render_-nya. Ini artinya React akan melompati proses _rendering_ komponen, dan menggunakan hasil _render_ terakhir.
```

Comment:
```suggestion
Apabila _function component_ Anda me-_render_ hasil yang sama jika diberikan _props_ yang sama, Anda dapat membungkusnya dengan menggunakan `React.memo` untuk meningkatkan kinerjanya dengan cara menyimpan (_memoize_) hasil _render_-nya. Ini artinya React akan melewati proses _render_ komponen, dan menggunakan hasil _render_ terakhir.
```

[Memoization](https://en.wikipedia.org/wiki/Memoization) is different from Memorization 😄